gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_27/01_change_api_description_datatype.yml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.5.27_change_api_description_datatype
+      author: GraviteeSource Team
+      changes:
+        - modifyDataType:
+            tableName: ${gravitee_prefix}apis
+            columnName: description
+            newDataType: nclob
```

Comment:
Are we sure NCLOB exists? it seems not listed https://docs.liquibase.com/reference-guide/change-types/column

Would it be worth running tests on different databases with existing data to ensure everything is functioning correctly?